spec/lib/engine/games/config_spec.rb

```diff
@@ -9,8 +9,13 @@ module Engine
   players = ('a'..'z')
 
   Engine::GAMES.each do |title|
+    min_p = players.take(Engine.player_range(title).min)
+
+    # 1824 2 player is only supported with Optional Rule Cisleithania activated
+    min_p = players.take(3) if title.to_s == 'Engine::Game::G1824'
```

Comment:
this doesn't seem to be the right approach

Reply:
@tobymao I suppose I should maybe assume that Cisleithania is always used if 2 players? So the optional rules is always active for 2.
The optional rule can dictate which flavor of 1824 you get for 3 players. 
And for 4-6 players I can ignore the optional rule.

This would mean the above code can be removed.

Also the 'Goods Times' optional rule, which is only relevant for 3-6 players, and not when using Cisleithania I guess can just be ignored.  
Can just do appropriate logging and remove the error checks for this.

Reply:
that's a good idea

Reply:
Ok, attempt done.